decodeEventLog.ts

```diff
@@ -4,7 +4,6 @@ import type {
   ExtractAbiEventNames,
   Narrow,
 } from "https://esm.sh/abitype";
-
```

Comment:
This empty line was to separate typical imports and partial hacky imports, what do you think about this?